### ISSUE TITLE
tries to resolve fsevents@1.2.4 but is long deprecated

### ISSUE THREAD

**Issue by cocoonkid** [checkly/headless-recorder#114]
```
> node install

node-pre-gyp WARN Tried to download(404): https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.4/fse-v1.2.4-node-v83-darwin-x64.tar.gz
node-pre-gyp WARN Pre-built binaries not found for fsevents@1.2.4 and node@14.3.0 (node-v83 ABI, unknown) (falling back to source compile with node-gyp)
```

Which version is recommended?

**Comment by maxigimenez** [comment]
@cocoonkid thanks for flagging this. I recommend removing `package-lock.json` locally until we update the one in the repo. This will fix the problem.

**Comment by maxigimenez** [comment]
For the record updating package-lock.json breaks our tests. But for local development works perfectly.

**Comment by maxigimenez** [comment]
This has been fixed with #120
